docs/overview/glossary.rst

```diff
@@ -21,19 +21,19 @@ All of these new words can get confusing! We've created a glossary to help nail
         E.g. Tensorflow and PyTorch.
 
     Ivy Transpiler
-        The transpiler allows framework to framework code conversions for supported frameworks.
+        The transpiler allows the framework to framework code conversions for supported frameworks.
```

Comment:
I guess the wording is right here
```suggestion
        The transpiler allows framework to framework code conversions for supported frameworks.
```

Reply:
I feel both are ok, but sure I will change it back.